src/pages/AssetPage/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import HeaderBar from '../../components/HeaderBar/HeaderBar';
+import { Container, Row, Col } from 'react-bootstrap'
+import { getUserAssets } from '../../api/assets-api';
+import { Asset } from '@imtbl/core-sdk'
+import AssetCard from '../../components/Card/AssetCard'
+import placeholderImg from '../../asset/placehoderImg.jpg'
+
+const AssetPage = () => {
+    const walletAddress = localStorage.getItem('WALLET_ADDRESS') || ''
+    const [userAssets, setUserAssets] = useState<Asset[]>([]);
+    const [noAssets, setNoAssets] = useState(false)
```

Comment:
I would avoid using one more state value here, I guess just a simple boolean variable will be enough for keeping `no-asset` state
```suggestion
    const noAssets = userAssets && userAssets.length === 0;
```